TESTS/mbedmicro-rtos-mbed/mutex/main.cpp

```diff
@@ -272,26 +261,23 @@ void test_single_thread_trylock(void)
     bool stat_b = mutex.trylock();
     TEST_ASSERT_EQUAL(true, stat_b);
 
-    osStatus stat = mutex.unlock();
-    TEST_ASSERT_EQUAL(osOK, stat);
+    mutex.unlock();
 }
 
 /** Test single thread lock
 
     Given a mutex and a single running thread
     When thread calls @a lock and @a unlock on the mutex
-    Then the returned statuses are osOK
+    Then @a lock and @a unlock operations are successfully performed.
 */
 void test_single_thread_lock(void)
 {
     Mutex mutex;
     osStatus stat;
 
-    stat = mutex.lock();
-    TEST_ASSERT_EQUAL(osOK, stat);
+    mutex.lock();
 
-    stat = mutex.unlock();
-    TEST_ASSERT_EQUAL(osOK, stat);
+    mutex.unlock();
```

Comment:
I am not sure how are we testing lock/unlock success/failure here? Should we modify the test to check from another thread if lock was applied or not and set success/fail based on that.

Reply:
ASSERT added in lock/unlock will force the test to fail in case of failure. 
I don't think that we need here to use another thread to show that lock/unlock works since we got other test cases which proves that.